library/src/scripts/headers/titleBarStyles.ts

```diff
@@ -92,6 +92,10 @@ export const titleBarVariables = useThemeCache(() => {
         },
     });
 
+    const titleBarNavAlignment = makeThemeVars("titleBarNavAlignment", {
```

Comment:
You can remove the "titleBar" part, it's redundant, because it's already scoped in "titleBar". Look at the class generated, it'll be "titleBar-titleBarNavAlignment", i'd rather get "titleBar-navAlignment"

Reply:
done